my_game.py

```diff
@@ -82,6 +85,13 @@ def __init__(self):
 
         self.center_x = random.randint(0, SCREEN_WIDTH)
         self.center_y = random.randint(0, SCREEN_HEIGHT)
+        self.change_x = random.randint(-ASTEROIDS_SPEED, ASTEROIDS_SPEED)
+        self.change_y = random.randint(-ASTEROIDS_SPEED, ASTEROIDS_SPEED)
+        if self.change_x == 0:
```

Comment:
That's not a speed of ASTEROIDS_SPEED. You need to use cos ans sin. Maybe there are functions for it in Arcade (The library).